docs/source/conf.py

```diff
@@ -98,7 +98,7 @@
 copyright = u'2014-2015, Red Hat'
 
 version = VERSION
-release = 0
+release = '0'
```

Comment:
Nice catch!  Although I did not see any error coming out of the docs build, did you?